### PR TITLE
Account for all tokens at genesis

### PR DESCRIPTION
**Pull request by garious**
#### Problem

Genesis tokens don't add up to 500 million

#### Summary of Changes

* Replace the service distribution placeholder
* Fixed a bug where the same tokens were put in 2 pools
* Added tests to ensure totals don't change
* Identify remaining inconsistencies

Needed in a follow-up PR:
* Account for bootstrap leader tokens
* Account for rent exemption fees
* Remove lockups from the community pool accounts
* Replace the validators placeholder

cc: @danpaul000, @mvines

### CODE REVIEW

**Comment by garious** [comment]
@rob-solana, can you take this one from here?

File: genesis/src/genesis_accounts.rs [hunk 1]
```diff
@@ -569,6 +665,307 @@ pub const VALIDATOR_INFOS: &[ValidatorInfo] = &[
...
         commission: 0,
     },
+    ValidatorInfo {
+        name: "tangible equipment", // Placeholder name
+        node: "Fe5sLQAAT7RBT8mcH1AAGCbExJQcYxcwXvp1GjrGbvxs",
+        vote: "Dr8MkZZuvZVQJFKtjShZYEfg6n93sc1GxevqLnGss7FW",
+        node_sol: 500.0,
+        commission: 0,
+    },
+    ValidatorInfo {
```
Comment (mvines) [inline genesis/src/genesis_accounts.rs]: Note that I intentionally did not add the `v0.21` placeholder validators to `master`, and as it stands this PR will encounter merge conflicts when lands in master and mergify attempts a backport.  I recommend you rebase this PR on `v0.21` and land it in that branch first -- that's were we need it for Sunday night anyway.  Then manually bring it back to `master` early next week, while removing these placeholder validators

**Comment by garious** [comment]
@mvines, this look better for the master branch? Can do a v0.21 PR for the validator placeholders after this lands.

File: genesis/src/genesis_accounts.rs [hunk 2]
```diff
@@ -23,427 +23,111 @@ const UNLOCKS_BY_TENTHS_FOR_60_MONTHS: UnlockInfo = UnlockInfo {
...
+        name: "tidy impression",
+        staker: "6ne6Rbag4FAnop1KNgVdM1SEHnJEysHSWyqvRpFrzaig",
+        sol: 5_000_000.0,
     },
     StakerInfo {
-        name: "jagged doctor",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 711_258.64,
+        name: "unbecoming silver",
+        // TODO: staker: "42yapY7Vrs5jqht9TCKsPoyb4vDFYcPfRkqAP85NSAQ", WrongSize
```
Comment (mvines) [inline genesis/src/genesis_accounts.rs]: Can we track this TODO in the SSOT doc instead of here?

File: scripts/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[package]
```
Comment (mvines) [inline scripts/Cargo.toml]: heh competition for bash.  I like it

File: genesis/src/genesis_accounts.rs (hunk 2 above)
Comment (garious) [reply genesis/src/genesis_accounts.rs]: Yeah, I added a comment to that cell and will follow up with Jeff as well

**Pull request merged**